specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/network.json

```diff
@@ -143,6 +143,21 @@
       "description": "Reference to another subresource.",
       "x-ms-azure-resource": true
     },
+    "ChildResource": {
+      "properties": {
+        "id": {
+          "type": "string",
```

Comment:
`id` should be read-only, I think

Reply:
It wasn't readonly for existing network child resource, so I kept it same. I removed child resources completely.